src/structures/GuildAuditLogs.js

```diff
@@ -409,6 +424,14 @@ class GuildAuditLogsEntry {
         }
         break;
 
+      case Actions.STAGE_INSTANCE_CREATE:
+      case Actions.STAGE_INSTANCE_DELETE:
+      case Actions.STAGE_INSTANCE_UPDATE:
+        this.extra = {
+          channel: guild.client.channels.cache.get(data.options?.channel_id) || { id: data.options?.channel_id },
```

Comment:
```suggestion
          channel: guild.client.channels.cache.get(data.options?.channel_id) ?? { id: data.options?.channel_id },
```

Reply:
We can change it but then we will have to change the existing one too